src/define_plugin_sets.h

```diff
@@ -814,10 +814,10 @@ To create/register a plugin, you have to :
   #define USES_MODBUS
 #endif
 
-#if defined(USES_C081) || defined (USES_C002) || defined(USES_P029)
+#if defined(USES_C001) || defined (USES_C002) || defined(USES_P029)
   #define USES_DOMOTICZ
 #endif
 
-#if defined(USES_C082) || defined (USES_C005) || defined(USES_C006) || defined(USES_C014)
+#if defined(USES_C001) || defined(USES_C002) || defined (USES_C005) || defined(USES_C006) || defined(USES_C014)
```

Comment:
C001 does not use MQTT, but it does share some code with C002 since they both are made for Domoticz.

Reply:
ok, will change it back..